tests/conftest.py

```diff
@@ -36,6 +39,30 @@
 ]
 
 
+class CustomSnapshotExtension(AmberSnapshotExtension):
+    def matches(
+        self,
+        *,
+        serialized_data,
+        snapshot_data,
+    ) -> bool:
+        if isinstance(serialized_data, str) and IS_WINDOWS:
+            # To make Windows path to work with snapshots
+            serialized_data = serialized_data.replace("\\", "/")
+            # To make POSIX path snapshot work with Windows
+            serialized_data = serialized_data.replace("//", "/")
+            # To fix side effects of above replace change
+            serialized_data = serialized_data.replace("https:/", "https://")
+        return super().matches(
+            serialized_data=serialized_data, snapshot_data=snapshot_data
+        )
+
+
+@pytest.fixture()
+def custom_snapshot(snapshot):
```

Comment:
`os_agnostic_snapshot` or something like that? custom is way too generic, it doesn't convey why a test would use custom vs regular snapshots.